### PR TITLE
Merge duplicated explainations about QGIS server installation with Apache

### PR DESCRIPTION
**Pull request by benoitdm-oslandia**
Move training_manual/qgis_server/install.rst sections to server_manual/getting_started.rst 

Try to do the best merge by replacing less detailed content of server_manual/getting_started.rst with training_manual/qgis_server/install.rst. Add references in training_manual to server_manual.

See ticket #6031

Goal:

Ticket(s): #

- [X] Could be Backport to LTR documentation

### CODE REVIEW

**Comment by benoitdm-oslandia** [comment]
@DelazJ can you review this one pls?

File: docs/training_manual/qgis_server/install.rst [hunk 1]
```diff
@@ -60,199 +60,19 @@ This is a good thing, it tells you we're on the right track as the server is
 saying that we haven't asked for a supported service. We'll see later on
 how to make WMS requests.
 
-|FA| HTTP Server Configuration
+|hard| |FA| HTTP Server Configuration
```
Comment (DelazJ) [inline docs/training_manual/qgis_server/install.rst:63]: Thinking more about it, what about throwing away this section? A |FA| section is supposed to hel the reader walk through a set of steps. There's no step here. Also the main information is to follow a doc that is elsewhere (breaking somehow the training workflow), and if this is `hard` (I guess, it's also optional), hence for advanced users, we should just let them read the server manual.
Or do I misunderstand the need of this section?

**Review by DelazJ — COMMENTED** [review]
@benoitdm-oslandia I did a review but, from a content point of view, I'm incompetent to give any judgement here... So if any server user/dev is around...

**Comment by benoitdm-oslandia** [comment]
Thanks for the review. I had basically move the doc from training to getting started and I have not checked further!

I will handle your corrections asap!

File: docs/server_manual/getting_started.rst [hunk 2]
```diff
@@ -88,83 +88,129 @@ Install Apache and  `mod_fcgid <https://httpd.apache.org/mod_fcgid/mod/mod_fcgid
...
-    FcgidInitialEnv QGIS_AUTH_DB_DIR_PATH "/var/www/qgis-server/qgisserverdb/"
-    FcgidInitialEnv QGIS_AUTH_PASSWORD_FILE "/var/www/qgis-server/qgisserverdb/qgis-auth.db"
-    
-    <IfModule mod_fcgid.c>
-    # Longer timeout for WPS... default = 40
-    FcgidIOTimeout 120
-    FcgidMaxRequestLen 26214400
-    FcgidConnectTimeout 60
-    </IfModule>
+   # QGIS log (different from apache logs) see https://docs.qgis.org/testing/en/docs/user_manual/working_with_ogc/ogc_server_support.html#qgis-server-logging
```
[the remark this answers is not in the document]
Comment (benoitdm-oslandia) [reply docs/server_manual/getting_started.rst:118]: agreed!

File: docs/server_manual/getting_started.rst [hunk 3]
```diff
@@ -494,6 +540,9 @@ QGIS Server needs a running X Server to be fully usable, in particular for print
 On servers it is usually recommended not to install it, so you may use ``xvfb``
 to have a virtual X environment.
 
+If you're running the Server in Desktop then there's no need to install xvfb.
```
[the remark this answers is not in the document]
Comment (benoitdm-oslandia) [reply docs/server_manual/getting_started.rst:543]: agreed!

File: docs/server_manual/getting_started.rst [hunk 4]
```diff
@@ -88,83 +88,129 @@ Install Apache and  `mod_fcgid <https://httpd.apache.org/mod_fcgid/mod/mod_fcgid
...
+
+.. note::
+
+   Remember that both the :file:`myhost.conf` and :file:`/etc/hosts` files should
+   be configured for our setup to work.
+   You can also test the access to your QGIS Server from other clients on the
+   network (e.g. Windows or macOS machines) by going to their :file:`/etc/hosts`
+   file and point the ``myhost`` name to whatever IP the server machine has on the
+   network. You can be sure that that specific IP is not ``127.0.0.1`` as that's
+   the local IP, only accessible from the local machine.  On ``*nix`` machines the
```
[the remark this answers is not in the document]
Comment (benoitdm-oslandia) [reply docs/server_manual/getting_started.rst:208]: fixed!

File: docs/server_manual/getting_started.rst [hunk 5]
```diff
@@ -88,83 +88,129 @@ Install Apache and  `mod_fcgid <https://httpd.apache.org/mod_fcgid/mod/mod_fcgid
...
+we also need to setup the client system so that it knows who ``qgis.demo``
+is. We do that by adding ``127.0.0.1 qgis.demo`` in the
+`hosts <https://en.wikipedia.org/wiki/Hosts_%28file%29>`_ file. We can do it
+with ``sh -c "echo '127.0.0.1 qgis.demo' >> /etc/hosts"``.
+Replace ``127.0.0.1`` with the IP of your server.
+
+.. note::
+
+   Remember that both the :file:`myhost.conf` and :file:`/etc/hosts` files should
+   be configured for our setup to work.
```
[the remark this answers is not in the document]
Comment (benoitdm-oslandia) [reply docs/server_manual/getting_started.rst:203]: fixed!

File: docs/training_manual/qgis_server/install.rst (hunk 1 above)
Comment (benoitdm-oslandia) [reply docs/training_manual/qgis_server/install.rst:63]: ok, what if I let the section as it but I remove the `|hard|` `|FA|` tags.

File: docs/server_manual/getting_started.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (benoitdm-oslandia) [reply docs/server_manual/getting_started.rst:203]: fixed

File: docs/training_manual/qgis_server/install.rst (hunk 1 above)
Comment (DelazJ) [reply docs/training_manual/qgis_server/install.rst:63]: Sorry for the late feedback
Hmmm, OK let's just remove the FA? Is this something optional or you anyway have to go through this section for a working server?

Comment (benoitdm-oslandia) [reply docs/training_manual/qgis_server/install.rst:63]: Sorry for the late feedback ;)

This section is not optional for a working server. 

The main goal was to regroup duplicated section about server configuration in one place and to make crossrefs instead. I have not enough time to reflow the `docs/training_manual/qgis_server/install.rst` and do a better work. For now, just removing the FA tag may be the best solution.

**Comment by benoitdm-oslandia** [comment]
thanks @DelazJ

**Review by benoitdm-oslandia — COMMENTED** [review]
Thanks!

**Pull request merged**